src/Annotations/AbstractAnnotation.php

```diff
@@ -373,6 +373,9 @@ public function jsonSerialize()
                 } else {
                     $ref['nullable'] = $data->nullable;
                 }
+                if (isset($data->title) && $data->title !== $defaultValues['title']) {
```

Comment:
Doesn't this violate the 3.1.0 spec? There is code a few lines above that explicitly allows `summary` and `description` next to `$ref`, but nothing more.

https://spec.openapis.org/oas/v3.1.0.html#schemaObject

Reply:
YouYou are right. In OpenApi3. 1, when nullable = true, type should be array

Reply:
It is not to add title to $ref
![image](https://github.com/zircote/swagger-php/assets/28125259/faf08971-3f51-411d-b9fd-760f2f8893ed)

Reply:
Hmm, interesting. I am not sure the specs do specify that sort of thing.
If it is valid do we need to handle other properties too apart from `title`?

Reply:
I'm not sure, maybe the description and summary can also be included. Now they are added to ref

Reply:
I guess we could try but there are some changes needed:
* update from master - I think I removed the `defaultValue` code
* the logic to add the title needs to be in a 3.1.0 conditional path only

Reply:
ok

Reply:
Oh, I didn't mean we cannot do this, just that there are some more changes required. Or are you doing a new PR?

Reply:
I am not doing a new PR. Do you need to merge this PR? If you need, I will reopen it.

Reply:
Yeah, let's reopen. Those changes have to go 😔 bro the PR.
I should have some time in the next few days for some more detailed feedback.

Reply:
Ok, thinks